api/source/graphql/queries/DocumentQueries.ts

```diff
@@ -1,7 +1,17 @@
-import { GraphQLList } from "graphql";
+import { GraphQLList, GraphQLID, GraphQLObjectType } from "graphql";
 import { DocumentType } from "../../models/DocumentType";
+import { UserType } from "../../models/UserModel";
 import { db } from "../../database/database";
-import { DOCUMENT_TABLE_NAME } from "../../database/utils/database_constants";
+import { DOCUMENT_TABLE_NAME, USER_TABLE_NAME } from "../../database/utils/database_constants";
+import { getDocumentByUserId } from "../helpers/DocumentHelper";
+
+const userDocuments = new GraphQLObjectType({
+  name: "userDocuments",
+  fields: {
+    user: { type: UserType },
+    documents: { type: GraphQLList(DocumentType) },
+  },
+});
 
 export default {
   documents: {
```

Comment:
Weird graphql object. Query should have a `user_id` (of type `GraphQLID`)  filter, then return `GraphQLList(DocumentType)`.